lib/fly.js

```diff
@@ -258,8 +264,7 @@ Fly.prototype.start = function (tasks, options) {
  * @return {void}
  */
 Fly.prototype.clear = function (paths) {
-	paths = arrify(paths)
-
+	paths = [].slice.call(arguments)
```

Comment:
see my comment on line [88](https://github.com/bucaran/fly/pull/171/files/990f9592b59a0dec2341a3fe3fe19917e08b0076#diff-de1e46e0afedd37aac70965ed3018e01R88) of this file